baybe/kernels/__init__.py

```diff
@@ -1,5 +1,5 @@
 """Kernels for Gaussian process surrogate models."""
 
-from baybe.kernels.basic import MaternKernel
+from baybe.kernels.basic import MaternKernel, ScaleKernel
 
-__all__ = ["MaternKernel"]
+__all__ = ["MaternKernel", "ScaleKernel"]
```

Comment:
Just for consistency reasons
```suggestion
__all__ = [
    "MaternKernel",
    "ScaleKernel",
]
```